src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -25,6 +25,16 @@ define(["dojo/_base/declare", "framework/PluginBase"],
 
                 // Hide the print button until the identify feature has been used.
                 $(this.printButton).hide();
+
+                // Display a clickable image thumbnail
+                $(this.container)
+                    .append('<p>Click the thumbnail to see a sample modal image popup</p>')
+                    .append('<div class="identify-point-plugin-thumbnail-image"' +
+                            'style="cursor: pointer"></div>');
+
+                $('.identify-point-plugin-thumbnail-image')
+                    .append('<img src="sample_plugins/identify_point/FutureHabitat_c.jpg"' +
+                            'id="sample-plugin-thumbnail" width="300" height="195" />');
```

Comment:
I dropped the class name altogether and am going to use the `sample-plugin-thumbnail` id attribute instead.